proto/oracle/v1/query.proto

```diff
@@ -157,16 +202,23 @@ message QueryRequestSearchResponse {
 // QueryRequestPriceRequest is request type for the Query/RequestPrice RPC
 // method.
 message QueryRequestPriceRequest {
+  // Symbol represents currency unit
   string symbol = 1;
+  // AskCount represents number of validators allowed for fulfilling the request
   int64 ask_count = 2;
+  // MinCount represents number of validators required to fulfill the request
```

Comment:
same question as above

Reply:
Could you elaborate? Not sure if it is formatting or `validators`

Reply:
I mean formatting krub. I think it should be `for fulfilling` to be consistent with the above comment.